bitrise.yml

```diff
@@ -80,7 +80,18 @@ workflows:
     - deploy-to-bitrise-io@2:
         inputs:
         - deploy_path: "${BITRISE_SOURCE_DIR}/DerivedData.zip"
-        - pipeline_intermediate_files: "$BITRISE_XCRESULT_PATH:BITRISE_XCRESULT_PATH" 
+        - pipeline_intermediate_files: "$BITRISE_XCRESULT_PATH:BITRISE_XCRESULT_PATH"
+    - script@1.1:
```

Comment:
We would need to add this script before the swiftlint is called so that it uses the latest version.
Swiftlinting is happening in this step https://github.com/mozilla-mobile/firefox-ios/blob/5beb450745e73a4cc2d70ec77c42db7e2e94560e/bitrise.yml#L386
Also it would be nice if we find a way to show the version used in the swiftlint step to be sure we are using the one installed here...